test/acceptance/member_test.exs

```diff
@@ -12,6 +13,9 @@ defmodule Community.Acceptance.MemberTest do
     fill_in "member", "twitter_handle", with: "cyclops"
     fill_in "member", "dribbble_username", with: "cyclopsdesign"
     fill_in "member", "website", with: "http://www.yahoo.com"
+    fill_in "member", "website", with: "http://www.yahoo.com"
```

Comment:
Is this supposed to be here twice? If so, maybe extracting a function that explains it a bit better might be good